webauthn4j-core/src/main/java/com/webauthn4j/validator/WebAuthnAuthenticationContextValidator.java

```diff
@@ -124,62 +150,74 @@ public WebAuthnAuthenticationContextValidationResponse validate(WebAuthnAuthenti
                 authenticator
         );
 
-
-        /// Verify that the value of C.type is the string webauthn.get.
+        //spec| Step7
+        //spec| Verify that the value of C.type is the string webauthn.get.
         if (!Objects.equals(collectedClientData.getType(), ClientDataType.GET)) {
             throw new MaliciousDataException("ClientData.type must be 'get' on authentication, but it isn't.");
         }
 
-        // Verify that the value of C.challenge matches the challenge that was sent to the authenticator in
-        // the PublicKeyCredentialRequestOptions passed to the get() call.
+        //spec| Step8
+        //spec| Verify that the value of C.challenge matches the challenge that was sent to the authenticator in
+        //spec| the PublicKeyCredentialRequestOptions passed to the get() call.
         challengeValidator.validate(collectedClientData, serverProperty);
 
-        // Verify that the value of C.origin matches the Relying Party's origin.
+        //spec| Step9
+        //spec| Verify that the value of C.origin matches the Relying Party's origin.
         originValidator.validate(collectedClientData, serverProperty);
 
-        // Verify that the value of C.tokenBinding.status matches the state of Token Binding for the TLS connection over
-        // which the attestation was obtained. If Token Binding was used on that TLS connection,
-        // also verify that C.tokenBinding.id matches the base64url encoding of the Token Binding ID for the connection.
+        //spec| Step10
+        //spec| Verify that the value of C.tokenBinding.status matches the state of Token Binding for the TLS connection over
+        //spec| which the attestation was obtained. If Token Binding was used on that TLS connection,
+        //spec| also verify that C.tokenBinding.id matches the base64url encoding of the Token Binding ID for the connection.
         tokenBindingValidator.validate(collectedClientData.getTokenBinding(), serverProperty.getTokenBindingId());
 
-        // Verify that the rpIdHash in aData is the SHA-256 hash of the RP ID expected by the Relying Party.
+        //spec| Step11
+        //spec| Verify that the rpIdHash in aData is the SHA-256 hash of the RP ID expected by the Relying Party.
         rpIdHashValidator.validate(authenticatorData.getRpIdHash(), serverProperty);
 
-        // If user verification is required for this assertion, verify that the User Verified bit of the flags in aData is set.
+        //spec| Step13
+        //spec| If user verification is required for this assertion, verify that the User Verified bit of the flags in aData is set.
         if (authenticationContext.isUserVerificationRequired() && !authenticatorData.isFlagUV()) {
             throw new UserNotVerifiedException("Validator is configured to check user verified, but UV flag in authenticatorData is not set.");
         }
 
-        /// Verify that the User Present bit of the flags in authData is set.
+        //spec| Step12
```

Comment:
I think it might be better to do this step 12 after step 11.
But I've confirmed to do so in https://github.com/webauthn4j/webauthn4j/pull/169/files#diff-ffae812986817112a8c56fdbab93aef0R120.